<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/3700051448975567catple_wallpapers_9656.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/logo_hust.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="funct3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="funct1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>View all restaurant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Finding restaurant in BKX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="funct2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Find top nearest restaurant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/logo pfiev edit.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
